### DISCUSSION TITLE
design error report

### DISCUSSION THREAD

**Discussion by arviinmo** [community/community#31057] [Accessibility]
Hi GitHub developers,
today when I visited my GitHub page I saw the following design problem appearing:
![image](https://user-images.githubusercontent.com/70845998/187045049-d4dcf9ee-b806-4355-81bc-e37fe117c014.png)
this is not a big problem but i just wanted to report the problem so that if you see this message you can fix it. (it is worth mentioning that this problem would be gone if you refresh the page or go below the page and then come back to the top!)

**Comment by radar-blip** [comment]
This was mentioned here also:  #31001
I see the problem remains, but will probably be fixed soon enough.
Feel free to report it as a bug if you like. 👍🏻

**Comment by tenmark86** [comment]
it's fixed refresh your github page. @arviinmo 

![image_2022-08-29_224005940](https://user-images.githubusercontent.com/83606182/187284255-de160626-9a66-4515-b974-4d05c5f36173.png)

**Reply by arviinmo** [reply]
I still have this bug!

**Comment by wiggledom** [comment]
Please try again using Ctrl + Shit + R
Maybe that will be work
If that doesn't work, please remove that account browser and create new browser

**Comment by cehfisher** [comment]
Hi @arviinmo - I wanted to follow up on this stale issue. Have the recent theme changes resolved this issue for you? If so, could you please close the issue? If not, what additional changes are needed? Thanks for the info! (cc @queenofcorgis)
